drafts/2019-10-15-openliberty-with-scala.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Calculating Mandelbrot Set using OpenLiberty microservices implemented in Scala"
+date:   2018-10-15 20:00:00 -0000
+categories: blog
+author_picture: https://avatars1.githubusercontent.com/u/50119017
+author_github: https://github.com/krzsam
+seo-title: Calculating Mandelbrot Set using OpenLiberty microservices implemented in Scala - OpenLiberty.io
+seo-description: An example application to calculate a Mandelbrot Set using Microprofile/Open Liberty microservices implemented in Scala. The goal is to demonstrate technical possibility of using OpenLiberty with Scala and possibility of deploying such solution as Kubernetes services.
+blog_description: "An example application to calculate a Mandelbrot Set using Microprofile/Open Liberty microservices implemented in Scala. The goal is to demonstrate technical possibility of using OpenLiberty with Scala and possibility of deploying such solution as Kubernetes services."
+---
+
+= Using Open Liberty in a Scala based application
+Krzysztof Samplawski <https://github.com/krzsam>
+
+This blog shows how Open Liberty microservices can be implemented using Scala language and deployed onto Kubernetes cluster.
+The main focus of the project was to explore technical feasibility of integration Open Liberty and Scala.
+
+== Docker image creation
+The base Docker image for Open Liberty is already available on Docker Hub. Available documentation on https://openliberty.io/[openliberty.io] provides
+information how to use this Docker image when using Java and Maven toolset to create application image, and
+the challenge was to analyze and replicate the same behaviour of creating an image but using Scala and sbt toolset, with some additional
+sbt plugins as required.
+An existing example application created by Maven was analyzed to investigate an internal structure of an application WAR file, which was
+then used to correctly parametrize sbt-docker plugin.
+Details on creating and testing Docker image can be found here: https://github.com/krzsam/scala-openliberty-mandelbrot/blob/master/README-Docker.md[README-Docker.md].
+
+== Running on Kubernetes
+The application Docker image was deployed on Kubernetes cluster and all required deployments and services were created.
+The services worked correctly which showed no interoperability issues between Open Liberty and classes as compiled by Scala,
+all annotations defined on classes and methods were correctly retained in runtime and then correctly picked up and used by Open Liberty.
+Details can be found here https://github.com/krzsam/scala-openliberty-mandelbrot/blob/master/README-k8s.md[README-k8s.md].
```

Comment:
"Details can be found in the https://github.com/krzsam/scala-openliberty-mandelbrot/blob/master/README-k8s.md[project README]." (if you agree?)

Reply:
Rephrased the link text in similar way across 2 references in lines 26 and 32, hope it looks better now